docs/source/en/using-diffusers/inpaint.md

```diff
@@ -121,4 +320,247 @@ unmasked_unchanged_image = PIL.Image.fromarray(unmasked_unchanged_image_arr.roun
 unmasked_unchanged_image.save("force_unmasked_unchanged.png")
 ```
 
-Forcing the unmasked portion of the image to remain the same might result in some weird transitions between the unmasked and masked areas, since the model will typically change the masked and unmasked areas to make the transition more natural.
+## Chained inpainting pipelines
```

Comment:
question for @patrickvonplaten

I remember I did this PR https://github.com/huggingface/diffusers/pull/4819 to supports chaining inpainting with other pipelines but it's not obvious how though -  we need to create `masked_image_latents` from `image` and `mask` with an additional step no?

Reply:
If you change the mask then yes, but if you keep the same mask then I don't think it's necessary. 

One might want to always keep the same mask, but inpaint multiple times